problem3.cs

```diff
@@ -0,0 +1,63 @@
+using System.CodeDom.Compiler;
```

Comment:
Instead of problem3.cs, please name this after the name of the problem

Reply:
done changes specified